gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Wed Dec 14 12:53:56 CET 2016
+#Fri Mar 03 09:38:24 GMT 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.2.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-3.3-all.zip
```

Comment:
If I change it to 

    distributionUrl=https\://services.gradle.org/distributions/gradle-3.4-all.zip

then the build fails like so:

```* What went wrong:
A problem occurred configuring project ':plugin'.
> Could not resolve all dependencies for configuration ':plugin:_debugPublish'.
   > Exception thrown while executing model rule: PublishingPlugin.Rules#publishing(ExtensionContainer)
      > org/gradle/api/internal/component/Usage
```

Is there much advantage to upgrading beyond what Android Studio prompts you to do automatically?